modules/prefs.js

```diff
@@ -67,7 +67,7 @@ function PrefManager() {
   this.compose_in_tab = prefsService.getBoolPref("compose_in_tab");
   // This is a hashmap
   this.monospaced_senders = {};
-  for each (s in this.split(prefsService.getCharPref("monospaced_senders")))
+  for (s of this.split(prefsService.getCharPref("monospaced_senders")))
```

Comment:
That's a mistake on my side, and I'm surprised that the strict mode didn't catch that.